pkg/database/database.go

```diff
@@ -311,6 +311,23 @@ func ConnectDB(dsn string, slowThreshold time.Duration, poolMax int, reset bool,
 					return fmt.Errorf("error migrating database to fasttrack schema 8073e7e037e5: %w", err)
 				}
 
+			case "8073e7e037e5":
+				log.Info("Migrating database to fasttrack schema ed364de02645")
+				if err := DB.Transaction(func(tx *gorm.DB) error {
+					if err := tx.Migrator().CreateIndex(&Run{}, "RowNum"); err != nil {
+						return err
+					}
+					if err := tx.Migrator().CreateIndex(&Metric{}, "Iter"); err != nil {
+						return err
+					}
+					return tx.Model(&SchemaVersion{}).
+						Where("1 = 1").
```

Comment:
Why is this needed?

Reply:
This is a quirky way to tell `gorm` to one row we have in that table. See the [docs](https://gorm.io/docs/update.html#Block-Global-Updates) for more details.